cpp/src/arrow/flight/sql/example/sqlite_sql_info.cc

```diff
@@ -0,0 +1,223 @@
+                                               "PRAGMA",
+                                               "PRECEDING",
+                                               "PRIMARY",
+                                               "QUERY",
+                                               "RAISE",
+                                               "RANGE",
+                                               "RECURSIVE",
+                                               "REFERENCES",
+                                               "REGEXP",
+                                               "REINDEX",
+                                               "RELEASE",
+                                               "RENAME",
+                                               "REPLACE",
+                                               "RESTRICT",
+                                               "RETURNING",
+                                               "RIGHT",
+                                               "ROLLBACK",
+                                               "ROW",
+                                               "ROWS",
+                                               "SAVEPOINT",
+                                               "SELECT",
+                                               "SET",
+                                               "TABLE",
+                                               "TEMP",
+                                               "TEMPORARY",
+                                               "THEN",
+                                               "TIES",
+                                               "TO",
+                                               "TRANSACTION",
+                                               "TRIGGER",
+                                               "UNBOUNDED",
+                                               "UNION",
+                                               "UNIQUE",
+                                               "UPDATE",
+                                               "USING",
+                                               "VACUUM",
+                                               "VALUES",
+                                               "VIEW",
+                                               "VIRTUAL",
+                                               "WHEN",
+                                               "WHERE",
+                                               "WINDOW",
+                                               "WITH",
+                                               "WITHOUT"}))},
+      {SqlInfoOptions::SqlInfo::SQL_NUMERIC_FUNCTIONS,
+       SqlInfoResult(std::vector<std::string>(
+           {"acos",    "acosh", "asin", "asinh",   "atan", "atan2", "atanh", "ceil",
+            "ceiling", "cos",   "cosh", "degrees", "exp",  "floor", "ln",    "log",
+            "log",     "log10", "log2", "mod",     "pi",   "pow",   "power", "radians",
+            "sin",     "sinh",  "sqrt", "tan",     "tanh", "trunc"}))},
```

Comment:
I'm curious: any particular reason these are lowercase, unlike the other entries here?

Reply:
No particular reason! Changed to keep consistency